client/components/carousel.jsx

```diff
@@ -0,0 +1,169 @@
+            <h3 className="caro-text">Album Score: <span>10 For Sure!</span></h3>
+          </div>
+        </div>
+      </div>
+    );
+    const favoriteLists = favorites.map((favorite, index) => {
+      if (favorite.favoriteType === 'album') {
+        return (
+          <div key={favorite.favoriteId} className={index === this.state.index ? '' : 'hidden'}>
+            <div className="caro-content-holder">
+              <div className="col-two-fifth caro-pic-col">
+                <h3 className="mg-t-75">{favorite.favoriteDetails.strAlbum}</h3>
+                <div className="caro-pic-holder">
+                  <img src={favorite.favoriteDetails.strAlbumThumb} />
+                </div>
+              </div>
+              <div className="col-two-fifth caro-info-col">
+                <div className="caro-detail-bt-row"><button className="caro-detail-button">Detail</button></div>
+                <h3 className="caro-text">Artist: <span>{favorite.favoriteDetails.strArtist}</span></h3>
+                <h3 className="caro-text">First Release: <span>{favorite.favoriteDetails.intYearReleased || 'N/A'}</span></h3>
+                <h3 className="caro-text">Genre: <span>{favorite.favoriteDetails.strGenre || 'N/A'}</span></h3>
+                <h3 className="caro-text">Style: <span>{favorite.favoriteDetails.strStyle || 'N/A'}</span></h3>
+                <h3 className="caro-text">Album Score: <span>{favorite.favoriteDetails.intScore || 'N/A'}</span></h3>
+              </div>
+            </div>
+          </div>
+        );
+      }
+      if (favorite.favoriteType === 'artist') {
+        return (
+          <div key={favorite.favoriteId} className={index === this.state.index ? '' : 'hidden'}>
+            <div className="caro-content-holder">
+              <div className="col-two-fifth caro-pic-col">
+                <h3 className="mg-t-75">{favorite.favoriteDetails.strArtist}</h3>
+                <div className="caro-pic-holder">
+                  <img src={favorite.favoriteDetails.strArtistThumb} />
+                </div>
+              </div>
+              <div className="col-two-fifth caro-info-col">
+                <div className="caro-detail-bt-row"><button className="caro-detail-button">Detail</button></div>
+                <h3 className="caro-text ">Alternate Name: <span>{favorite.favoriteDetails.strArtistAlternate || 'N/A'}</span></h3>
+                <h3 className="caro-text">Born: <span>{favorite.favoriteDetails.intBornYear || 'N/A'}</span></h3>
+                <h3 className="caro-text">Genre: <span>{favorite.favoriteDetails.strGenre || 'N/A'}</span></h3>
+                <h3 className="caro-text">Style: <span>{favorite.favoriteDetails.strStyle || 'N/A'}</span></h3>
+                <h3 className="caro-text artist-web">Artist Website: <span>{favorite.favoriteDetails.strWebsite || 'N/A'}</span></h3>
+              </div>
+            </div>
+          </div>
+
```

Comment:
Try to make a custom `CarouselSlide` component for this to reduce some duplication. Inside the component you can use `props.children` to show only the custom parts of the card that are different with the artist/album. (Similar to PageContainer)

```jsx
<CarouselSlide key={} isHidden={} title={} image={}>
  <div className="caro-detail-bt-row"><button className="caro-detail-button">Detail</button></div>
  <h3 className="caro-text ">Alternate Name: <span>{favorite.favoriteDetails.strArtistAlternate || 'N/A'}</span></h3>
  <h3 className="caro-text">Born: <span>{favorite.favoriteDetails.intBornYear || 'N/A'}</span></h3>
  <h3 className="caro-text">Genre: <span>{favorite.favoriteDetails.strGenre || 'N/A'}</span></h3>
  <h3 className="caro-text">Style: <span>{favorite.favoriteDetails.strStyle || 'N/A'}</span></h3>
  <h3 className="caro-text artist-web">Artist Website: <span>{favorite.favoriteDetails.strWebsite || 'N/A'}</span></h3>
</CarouselSlide>
```